UI/src/authorizationpolicies/pointStateRoadAuthorizationPolicy.js

```diff
@@ -1,16 +1,12 @@
-(function(root) {
-  root.PointStateRoadAuthorizationPolicy = function() {
-    AuthorizationPolicy.call(this);
-
-    var me = this;
-
-    this.formEditModeAccess = function(selectedAsset) {
-      return ((me.isMunicipalityMaintainer() || me.isElyMaintainer() && me.hasRightsInMunicipality(selectedAsset.municipalityCode)) || me.isOperator()) && !me.isState(selectedAsset);
+(function (root) {
+    root.PointStateRoadAuthorizationPolicy = function () {
+        AuthorizationPolicy.call(this);
+        var me = this;
+        this.formEditModeAccess = function (selectedAsset) {
+            return ((me.isMunicipalityMaintainer() || me.isElyMaintainer() && me.hasRightsInMunicipality(selectedAsset.municipalityCode)) || me.isOperator()) && !me.isState(selectedAsset.get());
```

Comment:
Since the `me.isState()` needs to receive the `selectedAsset.get()`, shouldn't the `hasRightsInMunicipality `receive `selectedAsset.get().municipalityCode`

Reply:
selectedAsset.getMunicipalityCode()